segmentstore/storage/src/test/java/io/pravega/segmentstore/storage/chunklayer/UtilsWrapperTests.java

```diff
@@ -275,4 +284,191 @@ public void testInvalidParameters() throws Exception {
                 () -> wrapper.getExtendedChunkInfoList(null, false),
                 ex -> ex instanceof NullPointerException);
     }
+
+    /**
+     * Sanity test to check if a new chunk was created with the provided content.
+     * @throws Exception Throws exception in case of any error.
+     */
+    @Test
+    public void testSanityCreateChunk() throws Exception {
```

Comment:
I think it is better to use more descriptive names like
`testSanityCreateChunk `-> `testSanityWhenCreateChunkThrows`
that suggest that it is negative test.

I suggest we make similar changes to names of other tests that testSanity by throwing an exception.